docs/_includes/variation-content.html

```diff
@@ -1,34 +1,38 @@
 <section class="content_main">
 
-    <div class="component-header u-mb30">
-        <h1>
-            {{ page.title }}
-        </h1>
-    </div>
+    <h1 class="component-header u-mb30">
+        {{ page.title }}
+    </h1>
 
     {% if page.variation_groups and page.variation_groups.size > 0 %}
     <ul class="m-list m-list__horizontal u-mb30">
         {% for variation_group in page.variation_groups %}
         {% if variation_group.variation_group_name %}
         <li class="m-list_item">
-            <a class="m-list_link" href="#{{ variation_group.variation_group_name | slugify }}">{{ variation_group.variation_group_name }}</a>
+            <a class="m-list_link"
+               href="#{{ variation_group.variation_group_name | slugify }}">
+                {{ variation_group.variation_group_name }}
+            </a>
         </li>
         {% endif %}
         {% endfor %}
     </ul>
     {% endif %}
 
     {% if page.description %}
-        <section id="description" class="u-mb30" class="lead-paragraph">
```

Comment:
Not sure if this is intentional but this `lead-paragraph` class is making component's intro text real big.

<img width="719" alt="Screen Shot 2020-05-18 at 12 41 01 PM" src="https://user-images.githubusercontent.com/1060248/82238177-dc1c9100-9904-11ea-864e-0a391607307f.png">

Reply:
Interesting… all I was doing here was combining the erroneous `class="u-mb30" class="lead-paragraph"` into one class attribute. But looking at production it looks like if there are two class attributes, the second one is ignored. So… I guess we just delete that second `lead-paragraph` and `class` attribute?

Reply:
It's not a bug, it's a feature! Yeah just delete it.

Reply:
Removed in https://github.com/cfpb/design-system/pull/496/commits/c848894ec382e6f822110d5dfe915df00edc4538